components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -213,10 +213,6 @@ export interface GitpodServer extends JsonRpcServer<GitpodClient>, AdminServer,
     generateNewGitpodToken(options: GitpodServer.GenerateNewGitpodTokenOptions): Promise<string>;
     deleteGitpodToken(tokenHash: string): Promise<void>;
 
-    // misc
-    isGitHubAppEnabled(): Promise<boolean>;
```

Comment:
Is it backward compatible change to old dashboards?

Reply:
Old dashboard? As of this PR the dashboard won't use this, cf. https://github.com/gitpod-io/gitpod/pull/19070/commits/61d50ae1a6b85db6cb6931679b787128373a1580

Reply:
We don't control deployment order in Dedicated. There can be period of time when we already have new servers, but clients are still having old dashboard. Server APIs should be evolved in backward compatible manner because of it.

Reply:
This is a cold path on Dedicated anyways. 

Since we enabled Create Project Modal for all, the components would be calling the old API are not longer rendered.  It should be safe to drop. Same applies to the old Dashboard components for creating projects.

Reply:
ok, if you are sure it cannot break, trying to bring attention. On Cloud my understanding when we change server APIs in non backward compatible way, then we shoudl first deploy a client part, then only server.

Reply:
@akosyakov, I've added those method as no-op back again. Is that ok now?